docs/data/material/getting-started/templates/sign-in/getSignInTheme.tsx

```diff
@@ -0,0 +1,586 @@
+                '&:hover': {
+                  backgroundColor: alpha(brand[600], 0.3),
+                  borderColor: brand[700],
+                },
+              }),
+              ...(ownerState.variant === 'outlined' &&
+                ownerState.color === 'secondary' && {
+                  backgroundColor: alpha(gray[600], 0.1),
+                  borderColor: alpha(gray[700], 0.5),
+                  color: gray[300],
+                  '&:hover': {
+                    backgroundColor: alpha(gray[600], 0.3),
+                    borderColor: gray[700],
+                  },
+                }),
+              ...(ownerState.variant === 'text' && {
+                color: brand[200],
+                '&:hover': {
+                  backgroundColor: alpha(brand[600], 0.3),
+                  borderColor: brand[700],
+                },
+              }),
+            }),
+          }),
+        },
+      },
+      MuiIconButton: {
+        styleOverrides: {
+          root: ({ theme, ownerState }) => ({
+            ...(ownerState.size === 'small' && {
+              maxHeight: '32px',
+              maxWidth: '32px',
+            }),
+            ...(ownerState.size === 'medium' && {
+              height: '40px',
+              width: '40px',
+            }),
+            color: brand[600],
+            '&:hover': {
+              backgroundColor: alpha(brand[300], 0.3),
+              borderColor: brand[200],
+            },
+            ...(theme.palette.mode === 'dark' && {
+              color: brand[200],
+              '&:hover': {
+                backgroundColor: alpha(brand[600], 0.3),
+                borderColor: brand[700],
+              },
+            }),
+          }),
+        },
+      },
+      MuiDivider: {
+        styleOverrides: {
+          root: ({ theme }) => ({
+            borderColor: `${alpha(gray[200], 0.8)}`,
+            ...(theme.palette.mode === 'dark' && {
+              borderColor: `${alpha(gray[700], 0.4)}`,
+            }),
+          }),
+        },
+      },
+      MuiPaper: {
+        styleOverrides: {
+          root: ({ theme }) => ({
+            backgroundImage: 'none',
+            backgroundColor: gray[100],
+            ...(theme.palette.mode === 'dark' && {
+              backgroundColor: gray[800],
+            }),
+          }),
+        },
+      },
+      MuiDialog: {
+        styleOverrides: {
+          root: ({ theme }) => ({
+            '& .MuiDialog-paper': {
+              borderRadius: '10px',
+              border: '1px solid',
+              borderColor: theme.palette.divider,
+            },
+          }),
+        },
+      },
+      MuiCard: {
+        styleOverrides: {
+          root: ({ theme, ownerState }) => ({
+            backgroundColor: gray[50],
+            borderRadius: 10,
+            outline: `1px solid ${alpha(gray[200], 0.8)}`,
+            boxShadow: 'none',
+            transition: 'background-color, border, 80ms ease',
```

Comment:
Do we need `transition` on the card if it's a layout component (unless there's a hover effect, which doesn't seem to be the case!)?